src/main/java/com/ee/employeesportal/services/EmployeeService.java

```diff
@@ -0,0 +1,32 @@
+package com.ee.employeesportal.services;
+
+import com.ee.employeesportal.model.Employee;
+import com.ee.employeesportal.repositories.JpaEmployeeRepository;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
```

Comment:
For Service layer classes better to use `@Service` instead of `@Component`.

You don't need `@Data` for this class.

Reply:
okay.